.pre-commit-config.yaml

```diff
@@ -25,6 +25,7 @@ repos:
       - id: check-executables-have-shebangs
         exclude: '\.rs$' # This doesn't play well with #![allow(clippy::crate_in_macro_def)]
```

Comment:
Ah LOL is this one even needed? I might have just gotten the wrong one?

Reply:
I think it was the wrong one. Let me remove yours.